### PR TITLE
Reader: es6ify the feed-stream

### PR DESCRIPTION
**Pull request by blowery**
Prep for dismissable recs.
No feature changes.

### CODE REVIEW

**Comment by blowery** [comment]
Part of #9674

File: client/lib/feed-stream-store/feed-stream.js [hunk 1]
```diff
@@ -120,64 +115,63 @@ assign( FeedStream.prototype, {
...
 		}
-		post = FeedPostStore.get( postKey );
+		const post = FeedPostStore.get( postKey );
 		return post && post._state !== 'error' && post._state !== 'pending' &&
 			post._state !== 'minimal';
-	},
+	}
 
-	selectNextItem: function( ) {
+	selectNextItem( ) {
```
Comment (samouri) [inline client/lib/feed-stream-store/feed-stream.js]: extra space

Comment (blowery) [inline client/lib/feed-stream-store/feed-stream.js]: nice, fixed

**Comment by samouri** [comment]
Code looks good.  now just need to test

File: client/lib/feed-stream-store/feed-stream.js [hunk 2]
```diff
@@ -294,29 +295,29 @@ assign( FeedStream.prototype, {
...
+	}
 
 	/**
 	 * If possible, returns the post metadata from a given response.
 	 * @param {object} postWrapper - response with post metadata
 	 * @returns {*} - null or post object
 	 * @private
 	 */
-	_getPostFromMetadata: function( postWrapper ) {
+	_getPostFromMetadata( postWrapper ) {
```
Comment (bluefuton) [inline client/lib/feed-stream-store/feed-stream.js]: Can we remove the initial `_` from the method name?

(mentioned in https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/javascript.md)

File: client/lib/feed-stream-store/feed-stream.js [hunk 3]
```diff
@@ -325,9 +326,9 @@ assign( FeedStream.prototype, {
 			return postWrapper;
 		}
 		return null;
-	},
+	}
 
-	_addXPost: function( postURL, xPostedBy ) {
+	_addXPost( postURL, xPostedBy ) {
```
Comment (bluefuton) [inline client/lib/feed-stream-store/feed-stream.js]: Can we remove the initial `_` from the method name here too?

**Comment by samouri** [comment]
Tested:
- [x] scrolled through a bunch of feeds/tags/lists/a8c and everything worked as expected

🚢

**Pull request merged**